tests/unit/test_ip.py

```diff
@@ -617,3 +618,25 @@ def test_ipaddress_network(data):
 @pytest.mark.parametrize("data", IS_CLASSFUL)
 def test_is_classful(data):
     assert ip.is_classful(**data["sent"]) == data["received"]
+
+
+def test_sort_list_cidrs():
+    sent = "10.0.10.0/24, 10.0.100.0/24, 10.0.12.0/24, 10.0.200.0/24"
```

Comment:
What is the expectation of `"10.0.0.0/24, 10.0.0.0/16, 10.0.0.0/18"`?

Does it make sense to convert this to `ip_sorted` and leave it on the user to re-add the subnet mask back?

Reply:
Hmmm, good point... I guess I would just expect it to have the CIDR twice in the returned string.

Reply:
I would expect it to just return the list of CIDRs in the right order, so should be the /16, /18, and /24 I'd guess, from largest to smallest?

Reply:
Just added a test validating this assumption.